File: MaWPhotos/src/main/java/us/mikeandwan/photos/ui/photos/IPhotoActivity.java
package us.mikeandwan.photos.ui.photos;


import java.util.List;

import us.mikeandwan.photos.models.Photo;


public interface IPhotoActivity {
    void addWork();

    void removeWork();

    void onApiException(Throwable throwable);

    Photo getCurrentPhoto();

    List<Photo> getPhotoList();
}

File: MaWPhotos/src/main/java/us/mikeandwan/photos/ui/photos/IPhotoActivity.kt
package us.mikeandwan.photos.ui.photos

import us.mikeandwan.photos.models.Photo

interface IPhotoActivity {
    fun addWork()
    fun removeWork()
    fun onApiException(throwable: Throwable?)
    val currentPhoto: Photo?
    val photoList: List<Photo>?
}